src/clj/game/core/ice.clj

```diff
@@ -331,7 +331,9 @@
         old-strength (get-strength ice)
         new-strength (ice-strength state side ice)
         changed? (not= old-strength new-strength)]
-    (when (rezzed? ice)
+    (when (if (installed? ice)
+            (rezzed? ice)
+            true)
```

Comment:
This will update the `:current-strength` of cards in the deck, which isn't correct. It should probably check if the ice is being encountered.